File: server/handler/request/comment.go
package request

// CreateCommentRequest 创建评论请求
type CreateCommentRequest struct {
	PostID  uint   `json:"post_id" binding:"required,min=1"`
	Content string `json:"content" binding:"required,min=1,max=500"`
	ParentID uint  `json:"parent_id" binding:"omitempty,min=1"`
}

// ListCommentsRequest 评论列表请求
type ListCommentsRequest struct {
	PostID uint `form:"post_id" binding:"required,min=1"`
	SearchRequest
}

// UpdateCommentStatusRequest 更新评论状态请求
type UpdateCommentStatusRequest struct {
	StatusRequest
}

File: admin/src/types/comment.ts
import { SearchRequest, StatusRequest } from './common'

// 评论实体接口
export interface Comment {
  id: number
  post_id: number
  content: string
  parent_id?: number
  created_at: string
  updated_at: string
  status: number
}

// 创建评论请求
export interface CreateCommentRequest {
  post_id: number
  content: string
  parent_id?: number
}

// 评论列表请求
export interface ListCommentsRequest extends SearchRequest {
  post_id: number
}

// 更新评论状态请求
export interface UpdateCommentStatusRequest extends StatusRequest {}